ManatiCR/vitaminau
# VIT-71: Google analytics.

# Task [VIT-71](https://manati.atlassian.net/browse/VIT-71)
Add Google Analytics in vitamina u page.

# Installation
- `ahoy composer install`
- `ahoy drush cr && ahoy drush cim -y`

# Test
- Go to the site at the following URL:
  - http://vitaminau.docker/admin/config/system/google-analytics
- Make sure that the "Web Property ID" is the same as the one specified in the tour ticket (UA-178403851-1): "[VIT-71](https://manati.atlassian.net/browse/VIT-71)".

# CA
- Make sure the module is installed and activated correctly
